gridstatus/ercot.py

```diff
@@ -1773,6 +1778,57 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+
+        # just so we can share parse logic
+        kwargs = dict(
+            report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+            date=date,
+            verbose=verbose,
+            extension="csv",
+        )
+
+        if date == "latest":
+            doc = self._get_document(**kwargs)
+            doc = [doc]
+        else:
+            doc = self._get_documents(**kwargs)
+
+        df = pd.concat([pd.read_csv(i.url, compression="zip") for i in doc])
+
+        def handle_dst(time, dst_flag):
+            # shift an hour forward to get new timezone
+            # shift back to display correct how with new tz
+            if dst_flag == "Y":
+                nt = time + pd.Timedelta(hours=1)
+                nt.dt.tz_localize(self.default_timezone, infer=True)
+                nt = time - pd.Timedelta(hours=1)
+            else:
+                return time
+
+        # ANCHOR - handle dst
+        df["SCEDTimeStamp"] = np.vectorize(handle_dst)(
+            df["SCEDTimeStamp"],
+            df["RepeatedHourFlag"],
+        )
+
+        return df
```

Comment:
to be consistent with how we name variables, we want the final column names to be "SCED Time Stamp" and "System Lambda". Drop repeated hour since the time stamp column is properly typed and has timezone info